applications/NXmx.nxdl.xml

```diff
@@ -858,17 +858,30 @@
                             <item value="rectangular"/>
                         </enumeration>
                     </field>
-
-
-                    <field name="incident_polarisation_stokes" recommended="true">
+                    <field name="incident_polarisation_stokes" type="NX_NUMBER" recommended="false"
```

Comment:
This change makes "incident_polarisation_stokes" required (see [here](https://nexusformat.github.io/definitions/classes/applications/NXmx.html#nxmx-entry-instrument-beam-incident-polarisation-stokes-field)). I guess it should have been changed to `optional="true"`.

Reply:
The release has already been tagged. Does this prevent you from validating files?